### ISSUE TITLE
Convert initialize-globals to one or more ES6 modules

### ISSUE THREAD

**Issue by samreid** [phetsims/chipper#971]
From https://github.com/phetsims/chipper/issues/912, we discussed transforming initialize-globals to one or more ES6 modules.  Or pruning it so that less work is done in that phase.  I'm not sure of the cost/benefit ratio, leaving unassigned for now.

**Comment by zepumph** [comment]
I agree! I believe that with progress over in https://github.com/phetsims/query-string-machine/issues/45, we are likely ready for this work to be done.
